frontend/testing/cypress/package.json

```diff
@@ -25,6 +25,6 @@
     "eslint:fix": "eslint \"src/**\" --fix",
     "prettier:check": "prettier -c src/**/*.js",
     "prettier:format": "prettier -w src/**/*.js",
-    "test:studio": "cypress run -b chrome -s 'src/integration/studio/*.js'"
```

Comment:
Hvorfor har du fjernet dette?

Reply:
Funket ikke når jeg kjørte cypress, jeg og Nina fjernet den for at den ikke blir spesifikk til chrome, hun sa fjern den og push med den PR